### PR TITLE
Gf simple onboarding

### PR DESCRIPTION
**Pull request by frechg**
If a user has gotten review from Hound they will not see this onboarding UI.

If a user has no active repos they will see:
screen shot 2015-03-06 at 12 19 19 pm

If a user has active repos but has not gotten review from Hound on any of those active repos, they will see:
screen shot 2015-03-06 at 12 30 03 pm

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -3,4 +3,8 @@ def avatar_url(user)
     gravatar_id = Digest::MD5::hexdigest(user.email_address.downcase)
     "https://www.gravatar.com/avatar/#{gravatar_id}"
   end
+
+  def display_onboarding?
+    current_user.repos.select { |repo| repo.builds.count > 0 }.length == 0
```
Comment (salbertson) [inline app/helpers/application_helper.rb]: What do you think about using `empty?`?

```
current_user.repos.select { |repo| repo.builds.count > 0 }.empty?
```

**Comment by croaky** [comment]
"Let's"

File: app/models/user.rb [hunk 2]
```diff
@@ -35,6 +35,10 @@ def has_repos_with_missing_information?
     repos.where("in_organization IS NULL OR private IS NULL").count > 0
   end
 
+  def has_active_repos?
+    self.repos.active.count > 0
```
Comment (salbertson) [inline app/models/user.rb]: Redundant `self`?

File: app/assets/stylesheets/_animations.scss [hunk 3]
```diff
@@ -59,3 +59,19 @@
     @include transform(scale(0.95));
   }
 }
+
+/////////////////////////////////////////////////////////////////////
```
Comment (gylaz) [inline app/assets/stylesheets/_animations.scss]: Is it normal for us to use comments like these?

File: config/locales/en.yml [hunk 4]
```diff
@@ -6,3 +6,19 @@ en:
...
+    step-two-title: "Open a GitHub pull request on the repo."
+    step-two-desc:
+      "Hound will review all GitHu pull requests that are
+      opened on an activate repo for violations of your style
+      guide."
+    step-three-title: "Hound comments on your pull request."
+    step-three-desc:
+      "If Hound finds any violations of your style guid he will
+      comment inline on the violation in your GitHub pull
+      request."
```
Comment (salbertson) [inline config/locales/en.yml]: Should we still to `_` convention for these?

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (gylaz) [inline app/helpers/application_helper.rb]: How about `repo.builds.count > 0` to `repo.builds.any?`

File: app/views/repos/_onboarding.haml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+%section.onboarding
+  .inner-wrapper
+    %h1 Lets keep that code in style:
+    %ol.onboarding-steps
+      - if current_user.has_active_repos?
+        %li.step
+          %h4 Hound is active on #{ current_user.repos.active.count } of your repos!
```
Comment (gylaz) [inline app/views/repos/_onboarding.haml]: Can we move this to translations file?

**Comment by salbertson** [comment]
Couple questions, looks good. :+1:

File: app/views/repos/_onboarding.haml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+          %p= t("onboarding.step-one-desc")
+        %li.step.current-step
+          %h4 Open a pull request on an active repo.
```
Comment (gylaz) [inline app/views/repos/_onboarding.haml]: Translation file?

File: config/locales/en.yml [hunk 7]
```diff
@@ -6,3 +6,22 @@ en:
...
   pending_status: "Hound is busy reviewing changes..."
   success_status: "Hound has reviewed all the changes!"
+
+  onboarding:
+    title: "Let's keep that code in style:"
+    step_one_title: "Activate Hound on a repo."
+    step_one_alt_title: "Hound is active on %{count} of your repos!"
+    step_one_desc:
+      "Hound watches for pull requests on all active repos.
+      Activating public repos cost $0 private repos cost $12."
```
Comment (croaky) [inline config/locales/en.yml]: "Active public repos cost $0. Active private repos cost $12."

File: config/locales/en.yml [hunk 8]
```diff
@@ -6,3 +6,22 @@ en:
...
+    step_two_title: "Open a GitHub pull request on the repo."
+    step_two_alt_title: "Open a pull request on an active repo."
+    step_two_desc:
+      "Hound will review all GitHu pull requests that are
```
Comment (croaky) [inline config/locales/en.yml]: "GitHub"

File: config/locales/en.yml [hunk 9]
```diff
@@ -6,3 +6,22 @@ en:
...
+      opened on an activate repo for violations of your style
+      guide."
+    step_three_title: "Hound comments on your pull request."
+    step_three_desc:
+      "If Hound finds any violations of your style guid he will
```
Comment (croaky) [inline config/locales/en.yml]: "guide"

Comment (croaky) [inline config/locales/en.yml]: "he" -> "it"

**Comment by Koronen** [comment]
@frechg this PR appears to have been merged as 48340caa77092acb7b2d99daca2766e5b0885d8e, yet it is still open. As @croaky pointed out, this diff (which is now in master) contains two typos and a questionable pronoun.

**Comment by frechg** [comment]
@Koronen Yup, opened a PR for fixing the typos here: https://github.com/thoughtbot/hound/pull/715

Closing this PR.

**Pull request closed without merging**